spring-security-modules/spring-security-core/src/main/java/com/baeldung/app/controller/TaskController.java

```diff
@@ -30,4 +42,66 @@
 
         return ResponseEntity.ok().body(tasks);
     }
+
+    /**
+     * Example of restricting specific endpoints to specific roles using @PreAuthorize.
+     */
+    @GetMapping("/manager")
+    @PreAuthorize("hasRole('ROLE_MANAGER')")
+    public ResponseEntity<Iterable<Task>> getAlManagerTasks()
+    {
```

Comment:
For consistency with this class and other Java code on the site, let's use the convention/standard of putting the opening brace at the end of the line rather than on a line by itself. Same applies to the other methods, not just this one :)

Reply:
@KevinGilmore Should be updated now